### PR TITLE
docs(writing-a-plugin): added details about schema-utils

### PR DESCRIPTION
**Pull request by anikethsaha**
Added [`schema-utils`](https://github.com/webpack/schema-utils) in `writing-a-plugin` page for schema validation,

**screenshot**

image

### CODE REVIEW

File: src/content/contribute/writing-a-plugin.md [hunk 1]
```diff
@@ -73,6 +73,34 @@ module.exports = {
 };
 ```
 
+
+Use [`schema-utils`](https://github.com/webpack/schema-utils) in order to validate the options being passed through the plugin option. Here is an example
```
Comment (montogeek) [inline src/content/contribute/writing-a-plugin.md]:
```suggestion
Use [`schema-utils`](https://github.com/webpack/schema-utils) in order to validate the options being passed through the plugin options. Here is an example
```

File: src/content/contribute/writing-a-plugin.md [hunk 2]
```diff
@@ -73,6 +73,34 @@ module.exports = {
 };
 ```
 
+
+Use [`schema-utils`](https://github.com/webpack/schema-utils) in order to validate the options being passed through the plugin options. Here is an example
```
Comment (montogeek) [inline src/content/contribute/writing-a-plugin.md]:
```suggestion
Use [`schema-utils`](https://github.com/webpack/schema-utils) in order to validate the options being passed through the plugin options. Here is an example:
```

Sorry, didn't catch it earlier

**Comment by anikethsaha** [comment]
cc @montogeek @EugeneHlushko

**Comment by montogeek** [comment]
Thanks!

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**